sst_platform_tools-llvm-toolset.yaml

```diff
@@ -0,0 +1,42 @@
+document: feedback-pipeline-workload
+version: 1
+data:
+  name: LLVM Toolset
+  description: Collection of packages providing and LLVM-based compiler toolchain.
+  maintainer: sst_platform_tools
+
+  packages:
+  - llvm
+  - llvm-devel
+  - llvm-googletest
+  - llvm-libs
+  - llvm-static
+  - llvm-test
+  - clang
+  - clang-devel
+  - clang-libs
+  - clang-tools-extra
+  - git-clang-format
+  - python3-clang
+  - lld
+  - lld-devel
+  - lld-libs
+  - lld-test
+  - libomp
+  - libomp-devel
+  - libomp-test
+  - compiler-rt
+  - lldb
+  - lldb-devel
+  - python3-lldb
+
+  unwanted_packages:
```

Comment:
This won't work here. You can either add it to the [view config](https://github.com/minimization/content-resolver-input/blob/master/configs/view-eln.yaml) or use the new [separate exclusion list (preferred)](https://github.com/minimization/content-resolver/blob/master/config_specs/exclusion.yaml).